docs/api/search.md

```diff
@@ -27,6 +27,10 @@ Sort criteria:
 * The sort criteria must start with the field name.
 * The default sort type is ascending. If multiple sort types are given the last is used.
 * With multi-sorts the first sort takes precedence.
+* If a property that does not exist is sorted on it is treated as None.
```

Comment:
Can you add some backticks around some of the keyworks like `None`, `DESC` and `ASC`?